.gitignore

```diff
@@ -0,0 +1,2 @@
+# .gitignore
```

Comment:
-  [ ] As written in the project requirements, kindly create a file named `test.md` not `text.md` like you currently have and add it here in your `.gitignore` file.